contracts/interfaces/tokens/ISBT.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+/**
+ * @notice The SBT module
+ */
+interface ISBT {
+    event Minted(address to, uint256 tokenId);
```

Comment:
Add comments for events

Reply:
Will add it in the next releases. We should probably refactor the comments